core/src/main/java/jenkins/model/item_category/NestedProjectsCategory.java

```diff
@@ -0,0 +1,38 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import jenkins.model.Messages;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * A generic {@link ItemCategory}.
+ *
+ * This category should be moved to cloudbees-folder-plugin short-term.
```

Comment:
When plugins as `cloudbees-folder` upgrade their baseline, they can use the new API about Item Categorization and this `ItemCategory` is able to provide directly by `cloudbees-folder`. I consider it is the right way.